mmedit/datasets/sr_folder_video_dataset.py

```diff
@@ -57,18 +78,43 @@ def __init__(self,
         self.lq_folder = str(lq_folder)
         self.gt_folder = str(gt_folder)
         self.num_input_frames = num_input_frames
+        self.ann_file = ann_file
         self.filename_tmpl = filename_tmpl
         self.metric_average_mode = metric_average_mode
 
         self.data_infos = self.load_annotations()
 
+    def _load_annotations_from_file(self):
+        self.folders = {}
+        data_infos = []
+
+        with open(self.ann_file, 'r') as fin:
+            for line in fin:
+                key, max_frame_num = line.strip().split(' ')
```

Comment:
use rsplit, and restrict the split num to 2.

This way, spaces in paths are supported.

Reply:
If using `.split(' ', 2)`, for `path 1/000.png 10`, it will become `['path', 1/000.png', '10']` Need to join it back if we want to support spaces in paths.

Reply:
the number is num of splits. So 1 split splits to 2 items

note the "r" in rsplit

Reply:
Oh I missed `rsplit`. It should work then.